src/pages/Stream/components/DeleteStreamModal.tsx

```diff
@@ -42,25 +43,33 @@ const DeleteStreamModal = () => {
 			centered
 			className={styles.modalStyle}
 			styles={{ title: { fontWeight: 700 } }}>
-			<TextInput
-				type="text"
-				label="Are you sure you want to delete this stream?"
-				onChange={handleInputChange}
-				placeholder={`Type the name of the stream to confirm. i.e. ${currentStream}`}
-				required
-				value={confirmInputValue}
-			/>
-			<Group mt={10} justify="right">
-				<Button onClick={onCloseModal} className={styles.modalCancelBtn}>
-					Cancel
-				</Button>
-				<Button
-					className={styles.modalActionBtn}
-					disabled={confirmInputValue === currentStream ? false : true}
-					onClick={handleDeleteStream}>
-					Delete
-				</Button>
-			</Group>
+			<Stack>
```

Comment:
Can't we use the same Modal for all the cases? Why keep this `DeleteStreamModal` separately?

Reply:
Sure, we can use it here. I updated the stream modal to use it while keeping this file alive to manage the states.